frontend/src/pages/staffform/LocationsWizardStep.vue

```diff
@@ -282,13 +282,17 @@ const cdsAccordionItemBeingtoggled = (event: any, address: Address) => {
   />
 
   <hr />
-  <h3>Additional locations</h3>
+  <h3 data-text="Additional locations">Additional locations</h3>
   <div class="frame-01" v-if="otherAddresses.length > 0" aria-live="off">
     <cds-accordion v-for="(address, index) in otherAddresses" :key="address.index">
-      <div :data-scroll="`address-${index + 1}-heading`" class="header-offset"></div>
+      <div
+        :data-text="`${address.locationName || 'Additional location'}`"
+        :data-scroll="`address-${index + 1}-heading`"
+        class="header-offset"
+      ></div>
       <cds-accordion-item
         :open="open[address.index]"
-        :title="locationNames[address.index] || 'Additional location'"
+        :title="address.locationName || 'Additional location'"
```

Comment:
Understood, but there was an issue where the value wasn't being properly updated, especially when adding and removing entries one after another

Reply:
Ok, maybe if you replace the data-text at line 289 to the same thing:
`locationNames[address.index] || 'Additional location'`
it could work.

I'm trying to test it here but for some reason I couldn't make the login step work yet.